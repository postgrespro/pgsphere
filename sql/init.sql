--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of pg_sphere.sql.
--
CREATE EXTENSION pg_sphere;
select pg_sphere_version();
