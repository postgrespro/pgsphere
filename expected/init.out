--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of pg_sphere.sql.
--
CREATE EXTENSION pg_sphere;
select pg_sphere_version();
 pg_sphere_version 
-------------------
 1.4.0
(1 row)

